holoviews/plotting/mpl/element.py

```diff
@@ -240,6 +244,38 @@ def _finalize_ticks(self, axis, dimensions, xticks, yticks, zticks):
             tick_fontsize = self._fontsize(f'{ax}ticks','labelsize',common=False)
             if tick_fontsize: ax_obj.set_tick_params(**tick_fontsize)
 
+    def _update_backend_opts(self):
+        plot = self.handles["fig"]
+
+        model_accessor_aliases = {
+            "figure": "fig",
+            "axes": "axis",
+            "ax": "axis",
+            "colorbar": "cbar",
+        }
+
+        for opt, val in self.backend_opts.items():
+            parsed_opt = self._parse_backend_opt(
+                opt, plot, model_accessor_aliases)
```

Comment:
Why limit this to the `model_accessor_aliases`, can we pass in `dict(self.handles, **model_accessor_aliases)` instead?

Reply:
Nvm, I see the handling below.